index.html

```diff
@@ -2,9 +2,153 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <title>Title</title>
+    <title>Rekupe Digital Agency</title>
+    <link rel="stylesheet" href="styles.css">
 </head>
 <body>
+<header class="outer-container header-container">
+        <nav class="header-navigation">
+            <ul>
+                <li><a href="#case-studies">Case Studies</a></li>
+                <li><a href="#services">Services</a></li>
+                <li><a href="#contact-us">Contact Us</a></li>
+            </ul>
+        </nav>
+
+        <div class="header-content">
+            <h1>Your website is an investment in the success of your business.</h1>
+            <p>We are here to help you succeed.</p>
+            <button type="button">Let's work together</button>
+        <span class="header-content-image">
+            <img src="assets/squiggle_line_stripe.svg" alt="Squiggle line stripe image">
+        </span>
+        </div>
+</header>
+
+<main>
+    <section class="outer-container introduction-container">
+        <div class="inner-container introduction-inner">
+        <h2>Rekupe is a digital agency in Los Angeles.
+            <br><br>
```

Comment:
We willen geen <br> element meer gebruiken. Dit is nu allemaal op te lossen met CSS